edit-post/assets/stylesheets/_z-index.scss

```diff
@@ -3,8 +3,8 @@
 // value is designed to work with).
 
 $z-layers: (
+	'.editor-block-list__block-edit:before': 0,
```

Comment:
Not certain why this change was required but it broke the PlainText inside blocks (code block for instance). It should be fixed by this though #6844